docs/chymyst05.md

```diff
@@ -2,7 +2,7 @@
 
 # Limitations of the Chemical Machine (and how to overcome them)
 
-While designing the “abstract chemistry” for our application, we need to keep in mind certain limitations of the chemical paradigm.
+While designing the chemical laws for our application, we need to keep in mind that the chemical paradigm limits what we can do (for our own good).
 
 1. We cannot detect the _absence_ of a given non-blocking molecule, say `a(1)`, in the soup.
 This seems to be a genuine limitation of the chemical machine paradigm.
```

Comment:
We can discuss this. I already have a `SmartPool` that is aware of blocking calls in reactions, via the `BlockingIdle()` method. If a reaction uses `BlockingIdle()`, one more thread will be made available in the pool. Not sure if this is what you had in mind.